Userland/Libraries/LibWeb/Page/Page.cpp

```diff
@@ -65,6 +65,8 @@ void Page::load(URL::URL const& url)
 
 void Page::load_html(StringView html)
 {
+    // FIXME: #23909 Figure out why GC threshold does not stay low when repeatedly loading html from the WebView
+    top_level_traversable()->vm().heap().collect_garbage();
```

Comment:
You can get to the heap with just `heap()` as `Page` itself is a `JS::Cell` :)

Reply:
right! that does look nicer.